source/pages/Home/Panel/ActivityPanel.tsx

```diff
@@ -5,14 +5,27 @@ import { useStore } from 'hooks/useStore';
 import { PanelList } from './components/PanelList';
 
 export const ActivityPanel: FC = () => {
-  const { activeAccount } = useStore();
+  const { activeAccount, activeNetwork, accounts } = useStore();
 
   return (
     <>
       <div className="p-4 w-full h-full text-white text-base bg-bkg-3">
-        {activeAccount &&
-        activeAccount.transactions &&
-        Object.values(activeAccount.transactions).length > 0 ? (
+        {activeNetwork.chainId === 1 || activeNetwork.chainId === 4 ? (
+          accounts[0].ethTransactions === [] ? (
+            <p className="flex items-center justify-center text-brand-white text-sm">
+              You have no transaction history.
+            </p>
+          ) : (
+            <PanelList
+              data={accounts[0].ethTransactions}
```

Comment:
for this kind of data which is rendered by network we should always use the activeAccount to get this data from, if we call accounts[0] we getting the data for the first account even if the first account is not the active one

```suggestion
              data={activeAccount.ethTransactions}
```